CMakeLists.txt

```diff
@@ -96,23 +88,26 @@ function(add_stratum_abseil_libs _TGT)
         absl::bad_optional_access
         absl::bad_variant_access
     )
+    if(absl_VERSION VERSION_GREATER_EQUAL 20230125)
```

Comment:
Is this coming from a security coding standard recommendation?

Reply:
No, this comes from experimenting with building against newer versions of the Stratum dependencies. We're going to have to do it sooner or later, possibly to deal with a CVE. I'm trying to get a head start.